1209/4DDF/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: CP Sapling Development Board Revision B
+owner: OakDevelopmentTechnologies
+license: MIT
+site: https://github.com/skerr92/odt-dev-boards
+source: https://github.com/skerr92/odt-dev-boards
```

Comment:
Please link to the specific board and also link to the software from there (the CP pull request or your branch.)

Reply:
I believe I got it how you requested, let me know if it's correct.